mllib/src/test/scala/org/apache/spark/mllib/clustering/KMeansSuite.scala

```diff
@@ -75,7 +75,7 @@ class KMeansSuite extends SparkFunSuite with MLlibTestSparkContext {
 
     // Make sure code runs.
```

Comment:
while we're here, can we make `var model = ...` into `val model = ...` and delete L74 `val center = ...` which is not used.